frontend/src/plugins/ai/components/ChatPanel.vue

```diff
@@ -114,7 +114,10 @@ const requestAI = async (query: string) => {
       ? "https://api.openai.com/v1/chat/completions"
       : openAIEndpoint.value + "/v1/chat/completions";
   const messages: OpenAIMessage[] = [];
-
+  const model =
+    openAIModel.value === ""
+      ? "gpt-3.5-turbo"
```

Comment:
We may extract this value as a pre-defined constant in a separate file (likely `common.ts`).